src/api-reference/14.order-object-definitions.md

```diff
@@ -0,0 +1,214 @@
+# 14. Order Object Definitions
+
+## 14.A Order Object Definition
+
+### 14.A.1 Overview
+
+The Order object.
+
+**uuid**
+Data type: [uuid]
+Required: n/a.
+Order id.
+
+**fulfilment_partner_identifier**
+Data type: [integer]
+Required: n/a.
+For future usage.
+
+**external_identifier**
+Data type: [string]
+Required: n/a.
+
+**order_date**
+Data type: [timestamp]
+Required: n/a.
+The order date.
+
+**language**
+Data type: [string]
+Required: n/a.
+For future usage.
+
+**account_id**
+Data type: [id]
+Required: n/a.
+The account id.
+
+**shop_id**
+Data type: [id]
+Required: n/a.
+The shop id.
+
+**type**
+Data type: [string]
+Required: n/a.
+For future usage.
+
+**price**
+Data type: [integer]
+Required: n/a.
+The total price of the order.
+
+**vat**
+Data type: [integer]
+Required: n/a.
+The total vat price of the order.
+
+**price_after_vat**
+Data type: [integer]
+Required: n/a.
+The total price including vat of the order.
+
+**status**
+Data type: [order_status]
+Required: n/a.
+The status of the order.
+
+**invoice_address**
+Data type: <ApiLink to="7_B">Address</ApiLink>
+Required: n/a.
+The invoice address of the order.
+
+**order_shipments**
+Data type: <ApiLink to="7_A">Shipment</ApiLink>
+Required: yes.
+The shipments for the order.
+
+**order_lines**
+Datatype: [array] of <ApiLink to="14_B">Order lines</ApiLink>
+Required: n/a.
+The order lines of the order.
```

Comment:
moet er ook niet een voorbeeld order bij? De POST is dus anders dan de GET

Reply:
Aangepast